Notation.Plugin.AzureKeyVault/Command/DescribeKey.cs

```diff
@@ -23,7 +22,7 @@ public DescribeKey(string inputJson)
             var request = JsonSerializer.Deserialize(inputJson, DescribeKeyRequestContext.Default.DescribeKeyRequest);
             if (request == null)
             {
-                throw new ValidationException(invalidInputError);
+                throw new ValidationException("Failed to parse the request for the plugin. Please contact the Notation developer to resolve the issue.");
```

Comment:
Are we able to print out the real error message that triggered this failure? Otherwise, given this error message, even a Notation developer might find it's hard to figure out what's going on.

Reply:
https://github.com/Azure/notation-azure-kv/blob/38be1cc281a8786e733ec5bd861f0bfdccabc925/Notation.Plugin.AzureKeyVault.Tests/Command/DescribeKeyTests.cs#L41-L49

This exception can only be triggered when `inputJson` is "null," as demonstrated in the test case above. There are no additional error messages to provide. Other errors will throw an exception and return directly without reaching this line.